src/main/java/com/algorand/algosdk/logic/StateSchema.java

```diff
@@ -12,10 +12,10 @@
 @JsonPropertyOrder(alphabetic = true)
 public class StateSchema implements Serializable {
     @JsonProperty("nui")
-    BigInteger numUint = BigInteger.ZERO;
+    public BigInteger numUint = BigInteger.ZERO;
 
     @JsonProperty("nbs")
-    BigInteger numByteSlice = BigInteger.ZERO;
+    public BigInteger numByteSlice = BigInteger.ZERO;
```

Comment:
You're using "ApplicationStateSchema", not "StateSchema", so this change probably isn't required.

Reply:
For some reason the txn class uses StateSchema:  https://github.com/algorand/java-algorand-sdk/blob/develop/src/main/java/com/algorand/algosdk/transaction/Transaction.java#L142

When I wanted to copy it to the ApplicationStateSchema I wasn't able to access these fields without setting them to public. 

If I'm missing something please let me know